### PR TITLE
Hide ACP search when no search plugin is installed

### LINKED ISSUE #2746: ACP navbar search issue
"Search the forum" option should not be displayed if no search plugin is activated. Displays a not found error if no search plugin is activated.

"Search the forum" option should open in a new tab, not in the ACP. Right now it opens in the ACP with breadcrumb, copyright and vanilla theme applied to the page. Clicking on "Home" in the breadcrumb, opens the forum homepage in the ACP (with vanilla theme applied and some weird formatting) and the browser address bar points to homepage instead of ACP.

### PR DESCRIPTION
**Pull request by yariplus**
And open forum results in a new tab. Fix #2746

### CODE REVIEW

**Comment by yariplus** [comment]
Um, window open might be better than the click event, i'm not sure why I used click.

**Comment by julianlam** [comment]
Typically, I bind a click event to the button, and a keydown event to the input box, both to the same handler. The keydown event only listens for keycode 13, of course.

**Comment by yariplus** [comment]
This is the click I'm speaking of:

```javascript
			if (href) {
				ajaxify.go(href.replace(/^\//, ''));
			}else{
				$(this).find('ul li:last-child a')[0].click();
			}

I think it should really be this:

```javascript
			if (href) {
				ajaxify.go(href.replace(/^\//, ''));
			}else{
				window.open(RELATIVE_PATH + '/search/' + input.val());
			}
```

**Comment by yariplus** [comment]
I fixed the merge conflicts, let me know if this looks good?

**Comment by barisusakli** [comment]
@psychobunny does acp search require a search plugin? I though it was a local search.

File: public/src/admin/modules/search.js [hunk 1]
```diff
@@ -37,9 +37,13 @@ define(function() {
...
 			var input = $(this).find('input'),
-				href = firstResult ? firstResult : RELATIVE_PATH + '/search/' + input.val();
+				href = firstResult;
 
-			ajaxify.go(href.replace(/^\//, ''));
+			if (href) {
+				ajaxify.go(href.replace(/^\//, ''));
+			}else{
+				if (input.val()) window.open(RELATIVE_PATH + '/search/' + input.val());
+			}
```
Comment (barisusakli) [inline public/src/admin/modules/search.js]: Can be written as 

```
if (href) {
    ajaxify.go(href.replace(/^\//, ''));
} else if (input.val()) {
     window.open(RELATIVE_PATH + '/search/' + input.val());
}
```

To match our styling guide.

**Comment by barisusakli** [comment]
We shouldn't hide the entire ACP search if a search plugin isn't installed we just need to disable the `search the forum for <xx>` bit.

**Pull request closed without merging**